src/osd/OSD.h

```diff
@@ -2082,12 +2082,12 @@ class OSD : public Dispatcher,
   void handle_pg_stats_ack(class MPGStatsAck *ack);
   void flush_pg_stats();
 
-  ceph::coarse_mono_clock::time_point last_sent_beacon;
+  utime_t last_sent_beacon;
```

Comment:
To avoid the potential expensive clock_time system call whenever it is possible. But I think it is unnecessary now since we have already moved the send_beacon() logic into the tick_without_osd_lock thread(which does not require the big osd_lock anymore) and we are planning to migrate to  the standards-provided time structures now.

All fixed and repushed.  Thanks for the review!